lib/archethic/contracts.ex

```diff
@@ -165,6 +167,37 @@ defmodule Archethic.Contracts do
     end
   end
 
+  def is_valid_contract?(
```

Comment:
I think we might improve the naming of this function, as it suggest the validation of the contract itself. (It's a but confusing).
But finally, the purpose is to validate the call or the execution of a contract based on some input (incoming transaction)